cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/patch/CloudCredentialPatchDTO.java

```diff
@@ -0,0 +1,48 @@
+package com.cloudesire.platform.apiclient.dto.model.patch;
+
+import com.cloudesire.platform.apiclient.dto.model.dto.DTO;
+
+import javax.validation.constraints.NotNull;
+
+public class CloudCredentialPatchDTO extends DTO
+{
+    @NotNull
+    private Action action;
+
+    @NotNull
+    private Object payload;
+
+    public CloudCredentialPatchDTO( Action action )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "payload" is marked "javax.validation.constraints.NotNull" but is not initialized in this constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2637)